### PR TITLE
Use absolute paths to files from tests

### PR DESCRIPTION
**Pull request by lachaib**
As discussed in #266 I'm separating refactor on tests from original PR

### CODE REVIEW

File: python/test/test_action_yml.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+project_root = pathlib.Path(os.path.dirname(__file__)) / '../..'
 
 
 class TestActionYml(unittest.TestCase):
 
     def test_composite_action(self):
-        with open('../../action.yml', encoding='utf-8') as r:
-            dockerfile_action = yaml.load(r, Loader=Loader)
+        with open(project_root / 'action.yml', encoding='utf-8') as r:
+            dockerfile_action =yaml.safe_load(r)
```
Comment (EnricoMi) [inline python/test/test_action_yml.py:14]: space after `=` missing

File: python/test/test_action_yml.py [hunk 2]
```diff
@@ -1,17 +1,20 @@
+import os
+import pathlib
 import unittest
 
 import yaml
-from yaml import Loader
+
+project_root = pathlib.Path(os.path.dirname(__file__)) / '../..'
```
Comment (EnricoMi) [inline python/test/test_action_yml.py:7]:
```suggestion
project_root = pathlib.Path(os.path.dirname(__file__)) / '..' / '..'
```

Comment (EnricoMi) [inline python/test/test_action_yml.py:7]:
```suggestion
project_root = pathlib.Path(__file__).parent.parent.parent
```

**Review by EnricoMi — CHANGES_REQUESTED** [review]
Love it!

File: python/test/test_action_yml.py (hunk 2 above)
Comment (lachaib) [reply python/test/test_action_yml.py:7]: I'm going for the suggestion with `.parent` for each 3 files

File: python/test/test_action_yml.py [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
     def test_composite_action(self):
-        with open('../../action.yml', encoding='utf-8') as r:
-            dockerfile_action = yaml.load(r, Loader=Loader)
+        with open(project_root / 'action.yml', encoding='utf-8') as r:
+            dockerfile_action = yaml.safe_load(r)
 
-        with open('../../composite/action.yml', encoding='utf-8') as r:
-            composite_action = yaml.load(r, Loader=Loader)
+        with open(project_root / 'composite/action.yml', encoding='utf-8') as r:
+            composite_action =yaml.safe_load(r)
```
Comment (EnricoMi) [inline python/test/test_action_yml.py:16]: there are many more places ...

File: python/test/test_junit.py [hunk 4]
```diff
@@ -7,6 +9,7 @@
 from publish.junit import parse_junit_xml_files, get_results, get_result, get_content, get_message, Disabled
 from publish.unittestresults import ParsedUnitTestResults, UnitTestCase, ParseError
 
+test_files_path = pathlib.Path(os.path.dirname(__file__)) / 'files'
```
Comment (EnricoMi) [inline python/test/test_junit.py:12]: more like this, also in `python/test/test_publish.py`:

```suggestion
test_files_path = pathlib.Path(__file__).parent / 'files'
```

**Review by EnricoMi — CHANGES_REQUESTED** [review]

File: python/test/test_action_yml.py (hunk 3 above)
Comment (lachaib) [reply python/test/test_action_yml.py:16]: Sorry it's getting late my 🧠  stopped for a while 😅

**Review by EnricoMi — APPROVED** [review]

**Comment by EnricoMi** [comment]
Thanks!

**Pull request merged**